### ISSUE TITLE
Some easier way to associate live guests

### ISSUE THREAD

**Issue by zusorio** [slmnio/slmngg-sfc#205]
Because names for live guests are sometimes outdated and don't always match their slmngg, battle.net, and sometimes discord it's a headache to set them up for matches. As this needs to be done pretty quickly we should make it easier

**Comment by slmnio** [comment]
I agree the system could do with being looked at. The idea for things like player cams is they are controlled by the player objects themselves through some UI and live guests continue to be just an extension of the player record.

One short-term thing we could do is reconfigure the live guests record. The `name` can instead be a lookup from the attached player record, and add an additional `name_override` for anything that needs something custom. That way, if you're interacting with live guests on Airtable, they'll follow the names of the player record associated.

**Comment by zusorio** [comment]
I like the name override idea a lot, should make stuff simpler in the short term

**Comment by slmnio** [comment]
![image](https://github.com/slmnio/slmngg-sfc/assets/15251071/0a995676-9ace-4bd8-9adc-a5e31f3048fd)
Okay I've made that change (was easier than I expected, the formula didn't give an array like a lookup so we're all good). This is live. By default, everyone has their override name set, so we'll have a period of crossover where we can update everyone.
